wa/framework/target/descriptor.py

```diff
@@ -153,10 +153,10 @@ def get_descriptions(self):
               ``True`` would suppress that, ensuring that only the base Target
               interface is initialized.
 
-              You may want to set this if there is a problem with one or more default
-              modules on your platform (e.g. your device is unrooted and cpufreq is
-              not accessible to unprivileged users), or if Target initialization is
-              taking too long for your platform.
+              You may want to set this to false if there is a problem with one
```

Comment:
"False" should be capitalized and enclosed in double grave quotes, as referring to a Python literal value.